tests/unit/test_textInfos.py

```diff
@@ -60,7 +60,6 @@ def test_surrogatePairsBackward(self):
 		obj = BasicTextProvider(text=u"\U0001f926\U0001f60a\U0001f44d") # 🤦😊👍
 		ti = obj.makeTextInfo(Offsets(5, 5))
 		ti.expand(textInfos.UNIT_CHARACTER) # Range at 👍
-		self.assertEqual(ti.offsets, (4, 6)) # Two offsets
```

Comment:
Was this intentional? Perhaps this could be replaced with another way to check the offsets?